docs/components/_clearpath_firmware.mdx

```diff
@@ -284,12 +304,21 @@ The Ridgeback's MCU is normally rotated 90 degrees when it is installed in the r
 Run the Clearpath firmware flash tool:
 
 ```
-source /opt/ros/humble/setup.bash
+source /opt/ros/jazzy/setup.bash
 ros2 run clearpath_firmware flash
 ```
 
 Select your platform, then select which method of flashing you would like to use.
 
+| Platform            | Flashing Method |
+| :------------------ | :-------------- |
+| Husky A300          | Ethernet        |
+| Jackal J100         | USB             |
+| Warthog W200        | USB or Ethernet |
```

Comment:
W200, R100, DX1x0 are all only USB for ROS 2.

Reply:
Updated.